task15

vramaniuk opened the pull request.

vramaniuk committed:
task15

AlexMokin commented on 1-select-tasks/22.txt:
```diff
@@ -1,3 +1,17 @@
 Create a SQL query to display the name of each customer along with their most expensive purchased product:
 
-| CompanyName | ProductName | PricePerItem |
```
One task per pull request!

AlexMokin commented on 1-select-tasks/15.txt:
```diff
@@ -1,3 +1,27 @@
+   [12] AS 'December' FROM(
+SELECT MONTH([o].[OrderDate]) AS [Months], COUNT(*) AS 'OrdersSum'
+FROM [Orders] [o]
+WHERE [o].[OrderDate] LIKE '%1997%'
```
use [year](https://docs.microsoft.com/en-us/sql/t-sql/functions/year-transact-sql) function

vramaniuk committed:
task15 [o].[OrderDate] LIKE "%1997%' replaced by YEAR([o].[OrderDate]) = 1997

vramaniuk committed:
task15

vramaniuk committed:
Revert "task15"

This reverts commit 1779eedc3551ff4e05b1161d2ad491fab58638a9.

vramaniuk committed:
task15

vramaniuk commented:
Sorry, I changed another file in a branch task15 accidentally
 and actually didn`t want to commit many solutions in one time. I fixed it. See af362f9 commit

AlexMokin commented:
well done!
